prov/usnic/src/usdf_ep_dgram.c

```diff
@@ -454,12 +454,16 @@ int usdf_dgram_fill_ep_attr(uint32_t version, struct fi_info *hints, struct
 	return FI_SUCCESS;
 }
 
-int usdf_dgram_fill_dom_attr(struct fi_info *hints, struct fi_info *fi)
+int usdf_dgram_fill_dom_attr(uint32_t version, struct fi_info *hints, struct fi_info *fi, struct usd_device_attrs *dap)
 {
 	struct fi_domain_attr defaults;
 
 	defaults = dgram_dflt_domain_attr;
-	defaults.name = strdup("usnic");
+
+	if (FI_VERSION_LE(version, FI_VERSION(1, 3)))
```

Comment:
It would be worth putting a comment here explaining why we are doing this.  Future Pierre (and Future Ben and Future Jeff) will thank Past Pierre for reminding us why this version comparison is here, and what the two different behaviors are (and why).